### ISSUE TITLE
Jailbreak fails at 24/33 repeatedly - Fails to install debs

### ISSUE THREAD

**Issue by kid1carus** [pwn20wndstuff/Undecimus#1106]
**Describe the bug**
Fails to install downloaded debs

**To Reproduce**
Rerun jailbreak

[Log File](https://pastebin.com/uzswrvs7)

**Device (please complete the following information):**
 - iOS Version: 12.1.2
 - iOS Device: iPad Mini 2
 - unc0ver Version: 3.2.1

**Place an "x" between the brackets if true:**
 - [ ] this is a bug others will be able to reproduce
 - [ x] this issue is present with all tweaks uninstalled(except for default packages) or disabled
 - [ x] this issue is present after a rootfs restore
 - [ x] this issue is present on the latest version of unc0ver

**Logs**
See log file link above

**Comment by NickogSo** [comment]
Hello, i’m french and i’m sorry for my bad english...

I’m not sure but i got maybe the same problem.

Cydia doesn’t want to open, so I tried to Re-Jailbreak but it makes me:

« Error (Fatal)
Errno: 9
Test: InstallDeb(dpkg_deb.UTF8String, true)
Filename: jailbreak.m
Line: 1274
Function: Jailbreak
Description: Unable to Install deb for dpkg. »

Is it the same ?
I need help too ☹️

**Comment by devnull75** [comment]
![onc0ver_error](https://user-images.githubusercontent.com/52005803/59780081-08580980-927f-11e9-8c92-d524cebbef96.PNG)

I have the same issue... iOS 11.3.1

**Comment by NickogSo** [comment]
The only one thing who worked for me was to erase and delete all contents... I couldn’t add or update any apps, launch Cydia, etc...

**Comment by TheRealKeto** [comment]
@kid1carus Seems as if unc0ver is trying to repair an essential package (APT 0.7 Transitional), but (somehow) fails due to the deb file for the package its trying to repair not being in the bundle of the current version of unc0ver that you have installed. 

This can be easily fixed by disabling every option and enabling the "SSH Only" option only, then attempting to jailbreak. Once you are jailbroken, open an SSH connection with a client and type the following command: 

`dpkg -i /var/lib/undecimus/apt/apt7_1:0-2_iphoneos-arm.deb`

After, simply reboot, and rejailbreak. (Please report back any further issues you may encounter)

**Comment by Benihipe** [comment]
Describe the bug
Fails to install debs while jailbreaking --> no jailbreak and cydia

To Reproduce
Rerun jailbreak

[https://pastebin.com/LiK9SMvW](url)

Device (please complete the following information):

iOS Version: 12.1.1
iOS Device: iPad Air 1
unc0ver Version: 3.2.1
Place an "x" between the brackets if true:

 this is a bug others will be able to reproduce
[ x] this issue is present with all tweaks uninstalled(except for default packages) or disabled
[ x] this issue is present after a rootfs restore
[ x] this issue is present on the latest version of unc0ver
Logs
See log file link above

Tryed to connect via ssh to ipad but couldn´t get a connection after enabling the "SSH Only so i couldn´t install the deb

**Comment by phoque1313** [comment]
This keeps happening to me too.  Tried M1 and M2, and jailbreaking with tweaks and/or daemons disabled.  I didn’t do restore rootfs because I’ve heard that it didn’t solve the problem for anyone else, so I’d rather just avoid doing that until I know whether or not I really need too.  Having SSH-only as the only option checked off seems to work, but it’s a bit useless to me now as I’m currently away from home and have no access to a computer.

iPhone 5s
iOS 12.1.1b3
Unc0ver 3.2.1

https://i.imgur.com/9i11pMa.jpg

**Comment by velkovb** [comment]
I get the same error even with having only SSH option. Tried restoring RootFS but get the same error.

iPhone 6S+
iOS 12.0.1
Unc0ver 3.2.1

![IMG_4646](https://user-images.githubusercontent.com/10958657/60756545-2b1c4900-a007-11e9-95dc-80429eb3b1d9.PNG)

I managed to do only SSH but I cannot connect to the phone over ssh to execute the command suggested. Also tried re-installing openssh but that did not help.

P.S. I ended up switching to Chimera and everything works fine there.
